src/backend/binaryen/utils.ts

```diff
@@ -1007,16 +1007,13 @@ export namespace FunctionalFuncs {
                 );
             }
             case ts.SyntaxKind.PercentToken: {
+                const emptyRef = binaryenCAPI._BinaryenRefNull(
```

Comment:
emptyRef is widely used, maybe we can make it as a global variable

Reply:
Yes, I will put it into utils